boards/arm/v2m_musca/v2m_musca-common.dtsi

```diff
@@ -22,6 +22,18 @@ dtimer0: dtimer@2000 {
 	interrupts = <5 3>;
 };
 
+mhu0: mhu@3000 {
```

Comment:
We should add `label` properties, its the current way we use in Zephyr to kinda map device_get_binding() and DTS.